### PR TITLE
Better mouse events handling

### PR DESCRIPTION
**Pull request by bdilly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## multi-seat-wayland
+
+If you have a laptop you could plug external mouse and keyboard
+and define a seat.
+
+You should add a rule to udev, something like:
+
+```sh
+sudo echo ENV{ID_VENDOR_ID}==\"046d\",ENV{ID_MODEL_ID}==\"c52b\",ENV{WL_OUTPUT}=\"HDMI1\",ENV{WL_SEAT}=\"whatever\" > /etc/udev/rules.d/98-wayland.rules
```
Comment (cabelitos) [inline README.md]: We can remove the WL_OUTPUT directive.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## multi-seat-vnc
+
+Just run the test program, it connect on default TCP port.
+
+```sh
+ $ ./multi-seat-vnc
+```
+
+Connect a local machine using default display ":0" and
```
Comment (cabelitos) [inline README.md]: local/remote

**Comment by cabelitos** [comment]
lgtm!

File: README.md (hunk 1 above)
Comment (bdilly) [inline README.md]: Oh, ok. I'll do it.

**Pull request merged**